src/transaction.js

```diff
@@ -304,4 +349,34 @@ export class Transaction {
       .toXDR()
       .toString('base64');
   }
+
+  /**
+   * Builds a FeeBumpTransaction
+   * @param {StrKey} feeSource - The account paying for the transaction.
+   * @param {string} fee - The maximum fee the account is willing to pay for this transaction.
+   * @param {TransactionEnvelope} innerTx - The Transaction to be bumped by this transaction.
+   * @param {string} networkPassphrase - networkPassphrase of the target stellar network (e.g. "Public Global Stellar Network ; September 2015").
+   * @returns {Transaction}
+   * @ignore tell jsdoc to not show this method for now
+   */
+  static buildFeeBumpTransaction(feeSource, fee, innerTx, networkPassphrase) {
```

Comment:
the fee parameter here is not consistent with `TransactionBuilder`. In `TransactionBuilder` you take a `baseFee` parameter and the compute the fee by multiplying `baseFee` with the number of operations. I think you should change `buildFeeBumpTransaction()` so that it takes a `baseFee` parameter and use that to calculate the `fee` set in the tx envelope.

I think you should also validate the `baseFee` parameter:

https://github.com/stellar/stellar-protocol/blob/master/core/cap-0015.md#validity
> The fee rate for E is at least the fee rate of F, otherwise return txINSUFFICIENT_FEE

Here are details on how the fee is calculated for fee bump transactions:

https://github.com/stellar/stellar-protocol/blob/master/core/cap-0015.md#fee-rate
> A fee-bump transaction has an effective number of operations equal to one plus the number of operations in the inner transaction. Correspondingly, the minimum fee for the fee-bump transaction is one base fee more than the minimum fee for the inner transaction. Similarly, the fee rate (see CAP-0005) is normalized by one plus the number of operations in the inner transaction rather than the number of operations in the inner transaction alone.

Reply:
@tamirms good point! I did this intentionally to let people calculate this value themselves. I'll add the validations :)

Reply:
updated